src/Adapters/Logger/FileLoggerAdapter.js

```diff
@@ -99,9 +99,12 @@ let _verifyTransports = ({infoLogger, errorLogger, logsFolder}) => {
 }
 
 export class FileLoggerAdapter extends LoggerAdapter {
-  constructor(options = {}) {
+  constructor(options) {
     super();
-
+    if (options && !options.logsFolder) {
+      throw "FileLoggerAdapter requires logsFolder";
```

Comment:
Yes it is, but unlikely to hurt anyone.
in index.js loggerAdapter is defaulted to undefined and logsFolder is the only option.
that would throw only if the user did `loggerAdatper: {}`